packages/api/bin/cli.js

```diff
@@ -85,9 +85,14 @@ program
 
 program
   .command('reset-tables')
+  .option('--username <username>', 'username [default: process.env.USERNAME]', process.env.USERNAME)
+  .option('--stack-name <stackName>', 'Name of stack', 'localrun')
+  .option('--system-bucket <systemBucket>', 'Name of systemBucket', 'localbucket')
+  .option('--no-repopulate', 'skip repopulating tables with test data. (for cleaning AWS tables)')
```

Comment:
I would just leave this out and only repopulate in test mode

Reply:
Like just pass in false to `resetTables`

Reply:
I just got rid of the option, and tested for testmode in serve.js.